src/route.js

```diff
@@ -0,0 +1,68 @@
+import express from 'express';
+import multer from 'multer';
+import fs from 'fs';
+import bodyParser from 'body-parser';
+import dotenv from 'dotenv';
+import InvertedIndex from '../src/inverted-index';
+
+dotenv.config();
+const app = express();
+const invertedIndex = new InvertedIndex();
+const upload = multer({ dest: 'fixtures/' });
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
All these can be in your server.js file then you import thre file here. It helps you seperate instances clearly. This file can contain only details that has to do with the routes while your server.js file has to do with your express.js setup for your project